ui/raidboss/data/05-shb/alliance/the_puppets_bunker.js

```diff
@@ -909,16 +909,15 @@
     },
     {
       'locale': 'ja',
-      'missingTranslations': true,
       'replaceSync': {
-        '724P-Operated Superior Flight Unit \\(A-Lpha\\)': '７２４Ｐ：強化型飛行ユニット［A-lpha］',
-        '767P-Operated Superior Flight Unit \\(B-Eta\\)': '７６７Ｐ：強化型飛行ユニット［B-eta］',
-        '772P-Operated Superior Flight Unit \\(C-Hi\\)': '７７２Ｐ：強化型飛行ユニット［C-hi］',
+        '724P-Operated Superior Flight Unit \\\\\\(A-Lpha\\\\\\)': '７２４Ｐ：強化型飛行ユニット［A-lpha］',
+        '767P-Operated Superior Flight Unit \\\\\\(B-Eta\\\\\\)': '７６７Ｐ：強化型飛行ユニット［B-eta］',
+        '772P-Operated Superior Flight Unit \\\\\\(C-Hi\\\\\\)': '７７２Ｐ：強化型飛行ユニット［C-hi］',
```

Comment:
I am not very confident about these regex lines.
Because in the txt file it is `724P-Operated Superior Flight Unit \(A-Lpha\)` ( with "\" before "(" )
[Regex101](https://regex101.com/) tell me that regex should be `724P-Operated Superior Flight Unit \\\(A-Lpha\\\)`
![image](https://user-images.githubusercontent.com/19927330/94422008-fe7ac100-01b8-11eb-9892-00b849c8aa5a.png)

If mine is right, then those regexes in DE/FR should be fixed.

Reply:
The `replacesync` regex is intended to match the text string present in the timeline file, and thus you are correct that both the `\` escape and `()` parentheses should be escaped.

Reply:
Great!
But I think it is better to separate a single pull request to fix DE/FR things.

Reply:
Yes, I think this is correct as well.  Sorry for how confusing this looks.  Other alternatives to consider in the future are `772P-Operated Superior Flight Unit .C-Hi.`